scripts/cleaning/delete-orgs.js

```diff
@@ -0,0 +1,49 @@
+require('isomorphic-fetch');
+require('abortcontroller-polyfill');
+
+const {homedir} = require('os');
+const {join} = require('path');
+const {config} = require('dotenv');
+const {getClient, yargGenerator, wasCreatedBefore} = require('./utils');
+config({path: join(homedir(), '.env')});
+
+function wasCreatedByTheCli(testRunId = '') {
+  return (key) => key.displayName?.startsWith(`cli-${testRunId}`);
```

Comment:
ish. that is risky. I created some orgs starting with `cli-`.
I would also check for the description:
`description === 'Generated by the Coveo CLI'`

Reply:
Organization doesn't have descriptions afaik, but because our ID start with id and ends up with g, I can do something like:
```suggestion
  return (key) => testRunId ? key.displayName?.startsWith(`cli-${testRunId}`) : key.displayName?.match(/cli-id.*g/);
```